packages/textkit/src/run/omit.js

```diff
@@ -5,11 +5,9 @@
  * @return {Object} run without ommited attribute
  */
 const omit = (value, run) => {
```

Comment:
Do you know what's different about the compiled versions of these two methods?

Reply:
TL;DR: `assign` modifies the object in place and calls setters vs spread operator creates a shallow copy.
Some interesting performance insights here: 
* https://www.reddit.com/r/node/comments/rp6jb8/discussion_object_spread_much_faster_than/

I _think_ we can safely replace assign with spread because nobody refers to "old" objects hoping that those are modified now?

Reply:
> I think we can safely replace assign with spread because nobody refers to "old" objects hoping that those are modified now?

Since both versions copy the objects, I think it safe to assume the old ones are discarded.  However, I have see a lot manipulation of objects in place in this repo, so I think we need to tread carefully in general.

> TL;DR: assign modifies the object in place and calls setters vs spread operator creates a shallow copy.
Some interesting performance insights here:

What I was trying to refer to is what does the code look like when it gets transpiled with babel?  I think it could be important to consider when working on micro-optimizations.

Also, maybe it could be worth checking out this codemod? https://github.com/babel/babel/tree/main/codemods/babel-plugin-codemod-object-assign-to-object-spread